### PR TITLE
FIX: Revert change to push notification excerpt

### PR DESCRIPTION
**Pull request by markvanlan**
I thought using `excerpt` would be much cleaner, but I forgot it's HTML.... woops

### CODE REVIEW

File: app/jobs/regular/create_chat_mention_notifications.rb [hunk 1]
```diff
@@ -48,7 +48,7 @@ def send_os_notifications(membership)
                                  username: @creator.username
                                 ),
         tag: DiscourseChat::ChatNotifier.push_notification_tag(:mention, @chat_channel.id),
-        excerpt: @chat_message.excerpt,
+        excerpt: @chat_message.message[0..399],
```
Comment (nattsw) [inline app/jobs/regular/create_chat_mention_notifications.rb:51]: I know this is a revert, but why 399?

Comment (ZogStriP) [reply app/jobs/regular/create_chat_mention_notifications.rb:51]: Same question 🤔 That number seems quite magical

Comment (markvanlan) [reply app/jobs/regular/create_chat_mention_notifications.rb:51]: don't question the magic

Comment (markvanlan) [reply app/jobs/regular/create_chat_mention_notifications.rb:51]: IDK why I had this before 🤔

Comment (nattsw) [reply app/jobs/regular/create_chat_mention_notifications.rb]: I think maybe defining why the number or what it means is nice.

Separately, would also be nice if `send_notifications` could be extracted to test?

Comment (markvanlan) [reply app/jobs/regular/create_chat_mention_notifications.rb]: There are some integration tests for chat message creation that run jobs immediately and test that push notifications are sent 🤷

Comment (nattsw) [reply app/jobs/regular/create_chat_mention_notifications.rb]: That's great!

File: app/models/chat_message.rb [hunk 2]
```diff
@@ -27,6 +27,10 @@ def excerpt
     PrettyText.excerpt(cooked, 50, {})
   end
 
+  def push_notification_excerpt
+    message[0...400]
```
Comment (nattsw) [inline app/models/chat_message.rb:31]: I think this is slightly better, we have still no idea why 400 tho. Given you've tried your best to explain but can't yet find anything better, let's let this go and please revisit if you suddenly 💡 

(Is it a char LIMIT?)

**Review by nattsw — APPROVED** [review]

**Pull request merged**